app/presenters/sufia/trophy_presenter.rb

```diff
@@ -18,6 +18,10 @@ def thumbnail_path
       solr_document[CatalogController.blacklight_config.index.thumbnail_field]
     end
 
+    def to_s
+      solr_document.title.first
```

Comment:
@awead I wonder if it would be better to just delegate to the solr_document. Curation concerns is already providing a to_s: https://github.com/projecthydra/curation_concerns/blob/master/app/models/concerns/curation_concerns/solr_document_behavior.rb